src/index.html

```diff
@@ -2,16 +2,203 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+      content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>HTML Form Task</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldSet">
+        <legend>Personal information</legend>
+        <div class="formField">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            required
+            id="surname"
+            autocomplete="off"
+              >
```

Comment:
Also 3rd item of checklist
```suggestion
          <input
            type="text"
            name="surname"
            required
            id="surname"
            autocomplete="off"
          >
```